scriptworker/monitoring/nagios_file_age_check.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+
+"""
+Checks for the existence and age of specific files, and that they are not
+too old.
+
+Allow 'old' to be defined as warning and critical arguments.
+
+You can either:
+- Specify one file to check
+- Give the path to a file, that contains a newline-separated list
+  of the files to check.
+
+This script will produce a multi-line nagios report in either case.
+
```

Comment:
The top level docstring should include attributes of the file: these are module-level variables.  For instance, `STATUS_CODE` would go here.
http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html